docs/finder-content-item.md

```diff
@@ -99,72 +99,195 @@ A string. Optional.
 Rendered in the header after the metadata. Can contain Govspeak and is rendered using the [Govspeak component](http://govuk-component-guide.herokuapp.com/component/govspeak).
 
 ## `facets`
-
 An array of hashes. Optional.
 
-Facets describe the metadata that the Finder deals with. They can contain several keys:
+Each facet corresponds to a document field that can take multiple values. Facets can be used to build user defined filters, and/or they can add metadata to search results.
 
-### `key`
+### Example: a filterable `Case type` facet
 
-A string. Required.
+```
+{
+  "key": "case_type",
+  "name": "Case type",
+  "type": "text",
+  "display_as_result_metadata": true,
+  "filterable": true,
+  "allowed_values": [...],
+  [...]
+}
+```
+<img alt="Filterable facets" src="./assets/filter.png" width="260" height="273">
```

Comment:
They changed the functionality, now you forced to use HTML tags instead if you need resizing http://stackoverflow.com/a/30151727